specification/assets/gherkin/metadata.feature

```diff
@@ -0,0 +1,25 @@
+@Metadata
+Feature: Metadata
+
+  Background:
+    Given a provider is registered
+
+  Scenario: Returns metadata
+    Given a Boolean-flag with key "metadata-flag" and a default value "true"
+    When the flag was evaluated with details
```

Comment:
```suggestion
    Given a Boolean-flag with key "metadata-flag" is evaluated with default value "false"
```

Let's use the same sentence structure as the [other steps](https://github.com/open-feature/spec/blob/d261f68331b94fd8ed10bc72bc0485cfc72a51a8/specification/assets/gherkin/evaluation.feature#L10), this way the same bindings will "just work" and less code needs to be implemented.

Reply:
I would specifically separate those two. Sometimes, you need to flag information without an evaluation. Hence, it is only a given state. But the when is the dedicated action. eg when it was modified. With this, we separate data and action. We already did the same thing within the testbed.